advisories/hackage/bzlib/HSEC-2024-0002.md

```diff
@@ -0,0 +1,58 @@
+```toml
+[advisory]
+id = "HSEC-2024-0002"
+cwe = [787]
```

Comment:
Just out of curiosity. Is there a CAPEC field?

Reply:
It's not in the schema, but we can add it.  I'll make a new issue for SRT to investigate that.

Reply:
CAPEC investigation issue: https://github.com/haskell/security-advisories/issues/158